src/Evolution/Systems/Cce/WorldtubeDataManager.cpp

```diff
@@ -29,28 +29,37 @@ MetricWorldtubeDataManager::MetricWorldtubeDataManager(
         buffer_updater,
     const size_t l_max, const size_t buffer_depth,
     std::unique_ptr<intrp::SpanInterpolator> interpolator,
-    const bool fix_spec_normalization) noexcept
+    const bool fix_spec_normalization,
+    const std::optional<double> extraction_radius) noexcept
```

Comment:
remove `const` and `std::move` the `extraction_radius` (or is that a trivial operation for a `std::optional<double>`?)

Reply:
it's trivial as far as I'm aware. std::optional stores a bool and the type